ceph.spec.in

```diff
@@ -776,6 +781,20 @@ well as the RESTful interface. These have to do with querying the daemons for
 command-description information, validating user command input against those
 descriptions, and submitting the command to the appropriate daemon.
 
+# python3-cmd2 and python3-colorama are not available in RHEL/CentOS, openSUSE
+# or SLES
```

Comment:
I assume this means it will not be available on Fedora derivatives? Would the only way to install be to use `pip`?

Reply:
@batrick won't unless the new-enough python3-cmd2 is packaged and shipped by distros.

@rpavani1998 instead, i'd suggest have a workaround for `with_argparser`, otherwise this tool is unlikely to be by our users.

Reply:
@tchaikov Will use an alternative instead. I think that can be implemented without using with_argparser.
Will work on it.